Tests/SourceKitLSPTests/SwiftInterfaceTests.swift

```diff
@@ -114,7 +114,9 @@ final class SwiftInterfaceTests: XCTestCase {
     _ testLoc: TestLocation, 
     ws: SKSwiftPMTestWorkspace, 
     swiftInterfaceFile: String, 
-    linePrefix: String
+    linePrefix: String,
+    file: StaticString = #filePath,
+    line: UInt = #line
```

Comment:
There are more XCT assertions here that should probably take line/file:
* XCTFail in the guard else
* XCTUnwrap called twice below